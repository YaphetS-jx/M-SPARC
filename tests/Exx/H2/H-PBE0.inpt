CELL: 20 20 20
MESH_SPACING: 0.3
BC: D D D
EXCHANGE_CORRELATION: PBE0
TOL_SCF: 1e-6
ELEC_TEMP_TYPE: fd
ELEC_TEMP: 315.773
MIXING_VARIABLE: potential
NSTATES: 2

PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1


