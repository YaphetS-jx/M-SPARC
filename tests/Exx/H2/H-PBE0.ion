ATOM_TYPE: H
PSEUDO_POT: H.pot
N_TYPE_ATOM: 2
COORD:
9 10 10
11 10 10