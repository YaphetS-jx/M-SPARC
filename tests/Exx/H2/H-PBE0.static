***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
      9.0000000000      10.0000000000      10.0000000000
     11.0000000000      10.0000000000      10.0000000000
Atomic forces (Ha/Bohr):
      0.0912002274      -0.0000000000      -0.0000000000
     -0.0912002274       0.0000000000       0.0000000000
