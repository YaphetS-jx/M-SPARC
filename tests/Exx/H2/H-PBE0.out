***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-Jan-2021  Start time: 16:44:37                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 20.000000 20.000000 
FD_GRID: 67 67 67
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999993
CHEB_DEGREE: 26
NSTATES: 2
EXCHANGE_CORRELATION: PBE0
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
MAXIT_FOCK: 20
TOL_FOCK: 1.00E-06
HYB_MIXING: 0.25
EXX_METHOD: FOURIER_SPACE
OUTPUT_FILE: ../tests/Exx/H2/H-PBE0.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.298507 (Bohr)
Output printed to                  :  ../tests/Exx/H2/H-PBE0.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  2
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.87 6.87 6.87
Number of atoms of type 1          :  2
Estimated total memory usage       :  1.17 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6374864373E-01        2.496E-01        7.139
2            -5.6555401907E-01        1.226E-01        8.631
3            -5.6592592743E-01        7.262E-02        7.596
4            -5.6608903332E-01        2.098E-02        6.942
5            -5.6609316903E-01        1.441E-02        7.243
6            -5.6609313293E-01        2.175E-03        6.175
7            -5.6609386432E-01        2.117E-03        6.361
8            -5.6609386879E-01        1.029E-03        5.731
9            -5.6609387077E-01        5.411E-04        5.108
10           -5.6609387141E-01        5.372E-05        4.336
11           -5.6609387141E-01        6.878E-05        3.972
12           -5.6609387181E-01        1.135E-05        3.663
13           -5.6609387060E-01        3.052E-06        2.961
14           -5.6609387101E-01        9.015E-07        2.173
Total number of SCF: 14    
No.1 Exx outer loop:
1            -6.3687775478E-01        6.528E-03        28.393
2            -6.3687787604E-01        4.567E-03        10.364
3            -6.3687796486E-01        2.944E-04        9.003
4            -6.3687796507E-01        2.395E-04        9.268
5            -6.3687796543E-01        6.388E-05        8.180
6            -6.3687796567E-01        1.928E-05        8.226
7            -6.3687796345E-01        1.102E-05        7.769
8            -6.3687796511E-01        2.678E-06        7.019
9            -6.3687796522E-01        1.070E-06        7.029
10           -6.3687796821E-01        5.910E-07        6.733
Total number of SCF: 10    
 Eex = 0.56640967
 Etot = -1.13215352
No.2 Exx outer loop:
1            -6.3693656094E-01        7.976E-04        27.404
2            -6.3693656410E-01        5.531E-04        9.204
3            -6.3693656525E-01        2.355E-05        8.137
4            -6.3693656463E-01        1.228E-05        8.489
5            -6.3693656449E-01        6.488E-06        7.727
6            -6.3693656435E-01        1.095E-06        8.137
7            -6.3693656449E-01        5.952E-07        6.660
Total number of SCF: 7     
 Eex = 0.56668141
 Etot = -1.13220278
No.3 Exx outer loop:
1            -6.3694671835E-01        9.511E-05        27.648
2            -6.3694671985E-01        6.550E-05        8.806
3            -6.3694671904E-01        1.916E-06        7.127
4            -6.3694671895E-01        1.453E-06        7.471
5            -6.3694671935E-01        1.003E-06        6.341
6            -6.3694671323E-01        3.872E-07        6.227
Total number of SCF: 6     
 Eex = 0.56672854
 Etot = -1.13221129
No.4 Exx outer loop:
1            -6.3694835747E-01        1.067E-05        27.992
2            -6.3694836187E-01        7.305E-06        7.516
3            -6.3694835913E-01        4.986E-07        6.590
Total number of SCF: 3     
 Eex = 0.56673609
 Etot = -1.13221270
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.6610634767E-01 (Ha/atom)
Total free energy                  : -1.1322126953E+00 (Ha)
Band structure energy              : -7.7417363708E-01 (Ha)
Exchange correlation energy        : -4.6635737400E-01 (Ha)
Self and correction energy         : -3.3160993387E+00 (Ha)
Entropy*kb*T                       : -1.1195413898E-09 (Ha)
Fermi level                        : -3.6978416206E-01 (Ha)
Average force                      :  9.1200227431E-02 (Ha/Bohr)
Maximum force                      :  9.1200227431E-02 (Ha/Bohr)
Time for force calculation         :  1.716 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  413.584 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
